pyews/soap.py

```diff
@@ -78,7 +79,8 @@ def send (self, request, debug=False):
         try:
             r = requests.post(self.url, auth=self.auth, data=request,
                               headers={'Content-Type':'text/xml; charset=utf-8',
-                                       "Accept": "text/xml"})
+                                       "Accept": "text/xml"},
+                              verify="/opt/odoo/authentication/java.pem")
```

Comment:
Cannot hardcode such a path which is outside of the package, and - worse - makes a reference to aonther unlreated package altogether (odoo)